Yosemite/Yosemite/SnapshotsProvider/FetchResultSnapshotsProvider.swift

```diff
@@ -189,6 +177,19 @@ public final class FetchResultSnapshotsProvider<MutableType: FetchResultSnapshot
             return nil
         }
     }
+
+    public func updateQuery(_ query: Query) {
```

Comment:
We're probably not gonna need this since the OrderListViewController will be recreated anyway but would you mind adding a comment on what happens if this is called? 🙂 Also, good choice about using a function instead of exposing the `query` property!

Reply:
yup this was removed in https://github.com/woocommerce/woocommerce-ios/pull/2897/commits/d6f8daf8ff384469d850a857ee718cfc090877ad 🙂